run.go

```diff
@@ -302,6 +308,15 @@ func actionRun(cc *cli.Context) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	engine.Collector = collector
 
+	// Send usage report, if we're allowed to
+	if opts.NoUsageReport.Valid && opts.NoUsageReport.Bool == false {
+		conn, err := net.Dial("udp", "k6reports.loadimpact.com:6565")
```

Comment:
No, there is no "connection" made, no handshaking procedure, with UDP communication. A simple test is to disable your network connection and do e.g. `k6 run http://localhost:6565`, which doesn't hang for me. The only thing that may hang is the DNS lookup. Would be nice to avoid that, but hard-coded IPs is of course a worse evil.